src/group_theory/perm/list.lean

```diff
@@ -303,6 +303,11 @@ begin
           ←nat.add_assoc] }
 end
 
+lemma form_perm_pow_apply_head (x : α) (l : list α) (h : nodup (x :: l)) (n : ℕ) :
```

Comment:
I feel like a `h : l.length > 0` hypothesis might be more convenient here.

Reply:
Not sure what you mean. How would one refer to the head element? If via nth_le, then that's exactly the lemma that is used in the convert statement.

Reply:
Nevermind, I misread this